.github/workflows/features.yml

```diff
@@ -17,7 +17,7 @@ name: feature tests
 # See: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
 on:
   pull_request:
-    branches: [master]
+    branches: [rmt_3]
```

Comment:
Wouldn't this break existing ruby2.5 CI runs?
Would it be possible to run CI on both the ruby2.5 if master and ruby3.2 if rmt_3?

Reply:
No, this would just create a new github action runs targeting the rmt_3 branch. Also the target branch is `rmt_3`